http/http.go

```diff
@@ -145,9 +186,8 @@ func (s *Server) newResponse(r *http.Request) (Response, error) {
 		ASN:        autonomousSystemNumber,
 		ASNOrg:     asn.AutonomousSystemOrganization,
 		Hostname:   hostname,
-		UserAgent:  userAgent,
 	}
-	s.cache.Set(ip, response)
+	s.cache.Set(response.IP, response)
```

Comment:
as far as I understood we need to revert everything in this block, just keep `newResponse()` as it was before

Reply:
:+1: